viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -15,7 +16,7 @@ vec3 packNormalToRgb( const in vec3 normal ) {
 }
 
 void updateFragmentColor(int renderMode, vec3 color, float treeIndex, vec3 normal, float depth, sampler2D matCapTexture) {
-    if (renderMode == RenderTypeColor) {
+    if (renderMode == RenderTypeColor || renderMode == RenderTypeEffects) {
```

Comment:
What am I missing here - why do we need to have this as a separate render mode? Seems like it's the same as `RenderTypeColor`?

Reply:
Usage is in update visibility shader